lib/components/Tag/Tag.tsx

```diff
@@ -3,29 +3,29 @@ import { MdAdd, MdClose } from 'react-icons/md';
 
 export interface TagProps {
   label: string;
-  onClick: () => void;
-  variant?: 'selectable' | 'added';
+  onClick?: () => void;
+  variant?: 'selectable' | 'added' | 'presentation';
```

Comment:
This can be left now like this, but it can bring bug later down when you forgot to add `onClick` for **selectable** or **added** variant as it is now optional when it should be required with those variants.
But as the component itself is so simple, this is just a minor annoyance.

Reply:
Added conditional types. If variant is `presentation`, onClick becomes `never`. Otherwise it's required and VS Code shows an error if it's missing.